src/debug/ee/debugger.cpp

```diff
@@ -14590,7 +14590,7 @@ void Debugger::SendRawLogMessage(
                  pAppDomain);
 
     ipce->FirstLogMessage.iLevel = iLevel;
-    ipce->FirstLogMessage.szCategory.SetString(pCategory->GetUnicode());
+    ipce->FirstLogMessage.szCategory.SetString(pCategory ? pCategory->GetUnicode() : NULL);
```

Comment:
This is only called from one place in `DebugDebugger::Log` that never passes in NULL. The null check is not necessary here.